framework/auth/views.py

```diff
@@ -251,6 +287,62 @@ def confirm_email_get(token, auth=None, **kwargs):
     ))
 
 
+@collect_auth
+def verified_email_remove(auth=None):
+    """Called at login if user cancels their merge or email add.
+    methods: DELETE
+    """
+    user = auth.user
+    confirmed_email = request.get_json()
```

Comment:
Minor: this seems like a misnomer. Reading through this view it, looks like this is a dict of the form `{'token': <some token>}`, not an email. How about just `json_body`?